code/datums/components/chem_booster.dm

```diff
@@ -277,6 +279,23 @@
 		meds_beaker.reagents.trans_to(wearer, 30)
 	setup_bonus_effects()
 
+///Updates the module on the armor to glow or not
+/datum/component/chem_booster/proc/update_module_icon(var/on_off)
+	var/obj/item/clothing/suit/modular/parent_armor = parent
```

Comment:
This should be setup as an early return in case the parent isn't modular armour, or the attachment is not in the slot, as components should are generic.